### PR TITLE
Introduce RISCV 64 experimental target

### PR DESCRIPTION
**Pull request by Hoverbear**
Just experimental support, untested. (#618 would do testing)

### CODE REVIEW

**Comment by Hoverbear** [comment]
bors try

**Comment by Hoverbear** [comment]
bors try

**Comment by Hoverbear** [comment]
bors try

**Review by Veykril — APPROVED** [review]

**Comment by Veykril** [comment]
bors merge

File: ferrocene/doc/user-manual/src/targets/index.rst [hunk 1]
```diff
@@ -173,6 +173,12 @@ should not be used in production.
      - Full
      - \-
 
+   * - :target:`riscv64gc-unknown-linux-gnu`
+     - ``aarch64-unknown-linux-gnu``
```
Comment (tshepang) [inline ferrocene/doc/user-manual/src/targets/index.rst:177]:
```suggestion
     - ``riscv64gc-unknown-linux-gnu``
```

Comment (Hoverbear) [reply ferrocene/doc/user-manual/src/targets/index.rst:177]: Ooops! Thank you!

**Review by Urhengulas — APPROVED** [review]
bors merge

**Pull request merged**